src/ui/dtl/actions/other/split_data/README.md

```diff
@@ -0,0 +1,61 @@
+# Split Data
+
+`Split Data` splits the input data into datasets based on condition.
+
+### Settings
+
+There are 4 ways of splitting the data:
+
+- **percent**: data will be split by the percentage of total images count.
+
+- **number**: data will be split by amount of images.
```

Comment:
Each dataset will contain selected number of images. The last split will contain fewer images if the total number of images isn't evenly divisible by the selected number of images per split.